packages/url_launcher/ios/Classes/UrlLauncherPlugin.m

```diff
@@ -67,11 +67,15 @@ - (instancetype)initWithViewController:(UIViewController *)viewController {
 }
 
 - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result {
-  NSString *url = call.arguments;
+  NSString *url = call.arguments[@"url"];
   if ([@"canLaunch" isEqualToString:call.method]) {
     result(@([self canLaunchURL:url]));
   } else if ([@"launch" isEqualToString:call.method]) {
-    [self launchURL:url result:result];
+    if (call.arguments[@"useSafariVC"]) {
+      [self launchURLInVC:url result:result];
+    } else {
+      [self launchURLInBrowser:url result:result];
```

Comment:
The URL might something else than HTML (mails to, and), so maybe this name is too web specific? handleInRegistredApp?

Reply:
Agreed. Changed to "launchURL".